changelog/v1.19.0-beta11/stat-name-label.md

```diff
@@ -0,0 +1,6 @@
+changelog:
+- type: NON_USER_FACING
+  description: >-
+   Allow using the label `~internal.solo.io/cluster-stat-name` to manually set
```

Comment:
```suggestion
   Allow using the label `internal.solo.io/cluster-stat-name` to manually set
```

Reply:
Just changed the actual label. I think we want the ~, otherwise any user in any namespace could break the metric for any service in any other namespace.